packages/launchpad-navigator/src/floating-navigator/index.tsx

```diff
@@ -8,9 +11,32 @@ export type FloatingNavigatorProps = {
 };
 
 const FloatingNavigator = ( { siteSlug }: FloatingNavigatorProps ) => {
+	const launchpadContext = 'navigator';
+	const checklistSlug = 'intent-build';
+
+	const {
+		data: { checklist },
+	} = useLaunchpad( siteSlug, checklistSlug );
+
+	const tasklistCompleted = checklist?.every( ( task: Task ) => task.completed ) || false;
+
+	const tracksData = { recordTracksEvent, checklistSlug, tasklistCompleted, launchpadContext };
+
+	const { setActiveChecklist } = useDispatch( LaunchpadNavigator.store );
+
+	const taskFilter = ( tasks: Task[] ) => {
+		return setUpActionsForTasks( {
+			tasks,
+			siteSlug,
+			tracksData,
+			extraActions: {
+				setActiveChecklist,
+			},
+		} );
+	};
```

Comment:
@valterlorran, this code repetition seems messy and probably unnecessary. Can you think of a way to make this behaviour the default? Or maybe create a wrapper component with these defaults?

Reply:
I added the component but didn't use it in the customer-home context. I think it will be better if we change it on a followup PR.

Reply:
I moved the logic to a new component called DefaultWiredLaunchpad